addons/point_of_sale/models/pos_session.py

```diff
@@ -417,11 +417,14 @@ def _accumulate_amounts(self, data):
 
             if order.is_invoiced:
                 # Combine invoice receivable lines
-                key = order.partner_id
-                invoice_receivables[key] = self._update_amounts(invoice_receivables[key], {'amount': order._get_amount_receivable()}, order.date_order)
+                key = order.partner_id.id
+                if self.config_id.cash_rounding:
```

Comment:
@simongoffin why changing this? Shouldn't you fix the _get_amount_receivable() function?